package.json

```diff
@@ -4,14 +4,52 @@
   "version": "0.0.1",
   "description": "The addon where ideas come to idea",
   "main": "index.js",
-  "author": "Les Orchard <lorchard@mozilla.com>",
+  "author": [
+    "Les Orchard <lorchard@mozilla.com>",
+    "Dave Justice <djustice@mozilla.com"
+  ],
   "engines": {
     "firefox": ">=38.0a1",
     "fennec": ">=38.0a1"
   },
-  "license": "MIT",
+  "scripts": {
+    "start": "st -nc .",
+    "package": "jpm xpi && mv @idea-town-addon-0.0.1.xpi dist/idea-town-addon-0.0.1.xpi"
+  },
+  "license": "MPL-2.0",
+  "dependencies": {
+    "exclude": "^1.0.0",
+    "xtend": "^4.0.0"
+  },
   "devDependencies": {
-    "gulp": "^3.9.0",
-    "gulp-connect": "^2.2.0"
+    "st": "^0.5.5"
+  },
+  "CONFIG": {
```

Comment:
Rather than treat package.json as a simple JSON config file, take a look at the [simple-prefs](https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/simple-prefs) API from the sdk. Simple-prefs lets you encode config values like these as preferences, so that we can manually edit them from `about:config`.